relay.config.js

```diff
@@ -2,7 +2,7 @@
 module.exports = {
   // ...
   // Configuration options accepted by the `relay-compiler` command-line tool and `babel-plugin-relay`.
-  src: "./src",
-  schema: "./data/schema.graphql",
-  exclude: ["**/node_modules/**", "**/__mocks__/**", "**/__generated__/**"],
-}
+  src: './src',
```

Comment:
the linter at work?

Reply:
Yes, I could set it to ignore config files however I've allowed it here